packages/helpwave_theme/lib/src/theme.dart

```diff
@@ -58,6 +58,7 @@ ThemeData makeTheme({
     bottomSheetTheme: BottomSheetThemeData(
       backgroundColor: backgroundColor,
       modalBackgroundColor: backgroundColor,
+      constraints: null,
```

Comment:
Is default null possible here?
```suggestion
      constraints: null,
```

Reply:
Yes, it means no constraints are used